src/pages/index.js

```diff
@@ -0,0 +1,282 @@
+    });
+    profileName.textContent = userData.name;
+    profileDescription.textContent = userData.about;
+    avatarImage.src = userData.avatar;
+  })
+  .catch((err) => {
+    console.error("Error fetching data:", err);
+  });
+
+const avatarProfileClosedBtn = document.querySelector(
+  ".profile__avatar-close-btn"
+);
+
+
+// profile form elements
+const profileEditButton = document.querySelector(".profile__edit-btn");
+const cardModalBtn = document.querySelector(".profile__add-btn");
+const profileName = document.querySelector(".profile__name");
+const profileDescription = document.querySelector(".profile__description");
+const editModal = document.querySelector("#edit-modal");
+const editFormElement = editModal.querySelector(".modal__form");
+const editProfileClosedBtn = editModal.querySelector(".modal__close-btn");
+const editModalNameInput = editModal.querySelector("#profile-name-input");
+const editDescriptionInput = editModal.querySelector(
+  "#profile-description-input"
+);
+
+//  delete   form  elements
+const deleteModal = document.querySelector("#delete-modal");
+const deleteForm = deleteModal.querySelector(".delete__form");
+const deleteModalClose = deleteModal.querySelector(".modal__close-btn");
+
+// card form elements
+const cardModal = document.querySelector("#add-card-modal");
+const cardForm = cardModal.querySelector(".modal__form");
+const cardSubmitBtn = cardForm.querySelector(".modal__submit-btn");
+const cardModalClosedBtn = cardModal.querySelector(".modal__close-btn");
+const cardTemplate = document.querySelector("#card-template");
+const cardsList = document.querySelector(".cards__list");
+const cardNameInput = cardModal.querySelector("#add-card-name-input");
+console.log(cardModal);
+const cardLinkInput = cardModal.querySelector("#add-card-link-input");
+
+// avatar form elements
+const avatarModal = document.querySelector("#avatar-modal");
+const avatarImage = document.querySelector(".profile__avatar-btn");
+const avatarForm = avatarModal.querySelector("#modal_form");
+const avatarSubmitBtn = avatarForm.querySelector(".modal__submit-btn");
+const avatarModalClosedBtn = avatarModal.querySelector(".modal__close-btn");
+const avatarInput = avatarModal.querySelector("#profile-avatar-input");
+
+const previewModal = document.querySelector("#preview");
+const previewModalImageEL = previewModal.querySelector(".modal__image");
+const previewModalCaptionEL = previewModal.querySelector(".modal__caption");
+function getCardElement(data) {
+  const cardElement = cardTemplate.content
+    .querySelector(".card")
+    .cloneNode(true);
+  const cardNameEL = cardElement.querySelector(".card__title");
+  const cardImageEL = cardElement.querySelector(".card__image");
+  const cardLikeBtn = cardElement.querySelector(".card__like-btn");
+  const cardDeleteBtn = cardElement.querySelector(".card__delete-btn");
+  cardNameEL.textContent = data.name;
+  cardImageEL.src = data.link;
+  cardImageEL.alt = data.name;
+
```

Comment:
* When I reload the page, all **red** hearts become **transparent** again.


Please, check `isLiked` and add `card__like-btn_liked` for the like button, if it's true